t/unicode-multibyte.t

```diff
@@ -0,0 +1,23 @@
+use lib '.';
+use t::Helper;
+use Mojo::File 'path';
+use Mojo::Util 'encode';
+use Test::More;
+
+my $builder = Test::More->builder;
+binmode($builder->output, ':encoding(UTF-8)');
+binmode($builder->failure_output, ':encoding(UTF-8)');
+binmode($builder->todo_output, ':encoding(UTF-8)');
```

Comment:
I'm surprised that you have to do this, since I've never done that in any other unicode related tests. Can you explain?

Reply:
This is just to avoid warnings when TAP is printing...for example:

````
t/unicode-multibyte.t .. 1/? Wide character in print at /opt/perl/perls/5.36.0/lib/5.36.0/Test2/Formatter/TAP.pm line 156.
Wide character in print at /opt/perl/perls/5.36.0/lib/5.36.0/Test2/Formatter/TAP.pm line 156.
t/unicode-multibyte.t .. ok
All tests successful.
Files=1, Tests=4,  0 wallclock secs ( 0.02 usr  0.01 sys +  0.23 cusr  0.02 csys =  0.28 CPU)
Result: PASS
0
````

I'm not sure it's the best way either, but I don't usually deal with output unicode from tests. I can remove it if you prefer.

Reply:
Sounds like validate_ok() should be fixed instead? If you think validate_ok() should be changed then please remove these three lines. If you want/can then please fix validate_ok() 😄

Reply:
I'm pretty sure validate_ok() is correct here, the messages it's outputting are perl strings and it's up to the tap formatter to deal with output encoding. I need to research how to run tests better with expected unicode output (checking mojo for example, test names explicitly use "\uNNNN" rather than the characters)...I think that's on me. For now, I've removed from the tests.